src/components/ModalDialog.js

```diff
@@ -19,10 +19,8 @@ export class ModalDialog extends React.Component {
         elementClass(document.body).remove('modal-backdrop-color-' + this.props.backdropColor);
     }
 
-    eventToOnHideArgs(e) {return e}
-
     onHide(e) {
-        this.props.onHide(this.eventToOnHideArgs(e));
+        this.props.onHide(e);
```

Comment:
Is this going to affect any other functionality? Why did we wrap`e` before?

Reply:
No. I used `eventToOnHideArgs` to avoid overriding `onHide` in `ThreejsEditorModal` classes which is no logner necessary.